interface1.html

```diff
@@ -900,1291 +903,1293 @@ <h3 style="margin-top: 15px; margin-bottom: -5px">Specify terminals
 		</fieldset>
 	</div>
 
-				<!--CONSTRAINTS-->
-				<div class="spotBlock">
-				<h2>Constraints</h2>
-				<fieldset>
+		<!--CONSTRAINTS-->
+		<div class="spotBlock">
+		<h2>Mapping constraints<div class="info"><span class="content">Mapping constraints refer to both syntax and prosody. They are similar to faithfulness constraints in that they regulate the relation between input and output (Prince & Smolensky 1993/2004, McCarthy & Prince 1995), but are distinct from faithfulness since syntactic and prosodic trees are built from different atoms (e.g. XPs and φs) as opposed to elements from the same alphabet (e.g. segments). Mapping constraints include constraints from competing theories, e.g. Match vs. Align/Wrap. They come in two types: syntax &rightarrow; prosody and prosody &rightarrow; syntax</span></div></h2>
```

Comment:
Not vital: I think we could clarify that SP constraints penalize syntactic features that don't translate to prosody and vice versa for PS constraints